### ISSUE TITLE
Strange error when connecting to bluemix dashdb on window7 64bit

### ISSUE THREAD

**Issue by nalleyda** [ibmdb/node-ibm_db#168]
Hi,
I've tried to connectino to bluemix dashdb on window7 64bit
Everytime I got same error like below. but it works on bluemix container.
## Here is error

C:{project dir}\node_modules\ibm_db\lib\odbc.js:119
  self.odbc = (options.odbc) ? options.odbc : ((ENV) ? ENV : new odbc.ODBC());
                                                             ^
 [object Object]

I'm using node v6.5.0 and v4.4.5 but both were shown same error.
There was no error when install npm install
---

var ibmdb = require('ibm_db');

ibmdb.open("{db.credential}, function (err,conn) {
  if (err) return console.log(err + " errorr1!!!!!");

  conn.query('select 1 from sysibm.sysdummy1', function (err, data) {
    if (err) console.log(err);
    else console.log(data);

```
conn.close(function () {
  console.log('done');
});
```

  });
## });

is there any solution about this error ?
Thanks.

**Comment by bimalkjha** [comment]
There seems some env issue in your system because this test code works fine for us. How to reproduce this issue? Thanks.

**Comment by bimalkjha** [comment]
Check with latest ibm_db@1.0.1. If issue still exist, reopen with repro steps. Thanks.

**Comment by alcmontejo** [comment]
@bimalkjha, We installed the version 1.0.1 however we are getting the same issue.

C:\<project dir>\node_modules\ibm_db\lib\odbc.js:145
self.odbc = (options.odbc) ? options.odbc : ((ENV) ? ENV : new odbc.ODBC());
^
[object Object]

**Comment by bimalkjha** [comment]
@alcmontejo let me know how can we reproduce this error? Also we need full error output. 
ls node_modules\ibm_db\installer\clidriver\lib
ls -l node_modules\ibm_db\build\Release

thanks.

**Comment by lubogregus** [comment]
had the same error: 
\node_modules\ibm_db\lib\odbc.js:145
self.odbc = (options.odbc) ? options.odbc : ((ENV) ? ENV : new odbc.ODBC());

this helped me:
go to ODBC and look under 'System DNS'. If you see following popup even though logged in as admin:

"You are logged in with non-Administrative privileges. System DSNs could not be created or modified"

try following link, I just started the other ODBC exe file from this link and once I did not see that popup entering 'System DNS', I created a new random ODBC connection profile and viola.. Error gone.

https://social.msdn.microsoft.com/Forums/sqlserver/en-US/5fd227fa-edca-4108-a4e8-71eb86928cf5/odbc-32bit-is-launching-64bit-odbc-administrator-no-menu-opens-32bit?forum=sqldataaccess

Good Luck.
